messenger.rst

```diff
@@ -730,6 +730,14 @@ options.
 AMQP Transport
 ~~~~~~~~~~~~~~
 
+.. versionadded:: 5.1
+
+    The AMQP transport has moved to a separate package. Install it by running
```

Comment:
I like this ... but I'd be more explicit in the message. The current proposal will only display `5.1` (because of the `versionadded` directive) and the message ... so we could reword it like this to be more clear about when this change will happen:

```rst
.. versionadded:: 5.1

    Starting from Symfony 5.1, the AMQP transport has moved to a separate package.
    Install it by running:
```